src/components/implementation/tests/unit_cbufmalloc/posix_test.c

```diff
@@ -0,0 +1,39 @@
+#include <printc.h>
+#include <cos_component.h>
+#include <cbuf_mgr.h>
+#include <assert.h>
+
+#define NUM 1024
+char *mptr[NUM];
+
+void
+cos_init(void *args)
+{
+	int err=0;
+
+	int i, j;
+
+	for (i = 0; i < NUM; i++) 
+	{
+		printc("Call to new malloc:\n");
+		mptr[i] = (char *)malloc((i+1) * sizeof(int));
```

Comment:
Testing calloc and realloc is a good idea. As to making sure I got a cbuf-backed memory, I see there is a cbuf_is_cbuf function, but it currently just returns false. Maybe in the future we can implement it?